cockroachdb/README.md

```diff
@@ -1,54 +1,30 @@
+## TL;DR
 
-## Deployment
+- [Which manifest should I choose?](#flavour)
+- [Example how to deploy Cockroach](example)
+- [Runbooks, how to test it manually, access the admin panel](#usage)
+- [Grafana dashboards](#metrics)
 
-To deploy a cockroachdb cluster in your namespace you will need to setup a `kustomization.yaml` file that will use the bases defined here with your own configuration layered over the top. There is an [example](./example/) folder that can be used as a starting point. By filling in the missing pieces (e.g. certs, backup config, etc) you should get a running CRDB cluster with periodic backups to S3 and AWS creds injected via [vault](https://github.com/utilitywarehouse/documentation/blob/master/infra/vault/vault-aws.md) (assumes an existing vault setup).
+## Flavour
 
-### Single namespace - multiple CockroachDB clusters
-
-While the preference is to have a single CockroachDB cluster per namespace, in some cases this isn't ideal. 
-
-An example of this is in the Energy team where they have recently split into three squads but currently still share use of the `energy-platform` namespace. 
-
-In order to deploy multiple CockroachDB clusters within a single namespace whilst avoiding naming conflicts we can make use of the `namePrefix` and/or `nameSuffix` ability of Kustomize. 
-This will automagically update the names of resources within a Kustomization as well as the selectors and labels. 
-
-One part of this that Kustomize is not able to help with is with the CockroachDB specific commands that are used to create and join nodes to the cluster. 
-For this two environment variables have been added `COCKROACH_INIT_HOST` and `COCKROACH_JOIN_STRING`. 
-
-- `COCKROACH_INIT_HOST` is used by the `init-job.yaml` manifest to initialise the first node in the cluster
-- `COCKROACH_JOIN_STRING` is in the `statefulset.yaml` manifest to define which nodes will be joining the cluster. 
-
-There are sensible defaults for these environment variables to ensure that a single cluster within a namespace can be brought up without overriding any environment variables. 
-
-
-### Versioning
-
-This repo uses tags to manage versions, these tags have two components:
+There are two manifest directories - `manifests-cfssl`, and `manifests-cert-manager`.
 
-  - The version of the `cockroachdb/cockroach` image the manifests are using
-  - An internal version to track changes to anything besides the version of
-    CockroachDB
+- `manifests-cfssl` are **not recommended** for fresh install - they contain the manifests that demand
+  certificate authority deployed within the namespace. For more information,
+  see [CFSSL README](manifests-cfssl/CFSSL_README.md)
+  - `manifests-cert-manager` are using Kubernetes cert-manager controller, 
```

Comment:
I think we should put `manifests-cert-manager` as the first entry, and call out that it is recommended, and call the cfssl one deprecated

Reply:
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR14)